src/js/__test__/imagenode.test.js

```diff
@@ -0,0 +1,137 @@
+import imagenode from '../imagenode';
+import options from '../options';
+
+describe('Image Node', () => {
+    let originalImage = (hasParams = true) => {
```

Comment:
it's best to put this in a `beforeEach`.
[](https://facebook.github.io/jest/docs/en/setup-teardown.html)